core_weekly/template.py

```diff
@@ -46,6 +50,15 @@ def headers(self):
 ## A quick update about PrestaShop\'s GitHub issues and pull requests:
 '''
 
+        result = template.replace("DDDD", today)
```

Comment:
Did not find how to perform multiple replacements so I did this not good looking code 😢

Reply:
You can use `format` instead of `replace` and name parameters instead as you can see here: https://github.com/PrestaShop/core-weekly-generator/pull/40/files#diff-51296f630d64d6414bc996571bd905b5R102

Reply:
Thank you ! Fixed in https://github.com/PrestaShop/core-weekly-generator/pull/40/commits/ff479af569d1e140b216f1e9333cfc8505a48587